src/nvim/pos.h

```diff
@@ -1,6 +1,9 @@
 #ifndef NVIM_POS_H
 #define NVIM_POS_H
 
+// for INT_MAX, LONG_MAX et al.
+#include <limits.h>
```

Comment:
unless this is required for things directly in pos.h, in general we don't want to have "mega-headers" that bring in other headers (`os.h` is an exception).

Reply:
It's used directly below it. Do you prefer moving the `MAXCOL`, `MAXLNUM` to another header to avoid an `#include`?

Reply:
No, but maybe worth mentioning in the commit message.

Reply:
`LONG_MAX` is in the patch description. Do you mean to mention that a new header is included in some file?

Reply:
No, I get it now.